File: Components/WinForms/VB/RichEditMainDemo/Modules/BulletsAndNumbering.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Drawing
Imports System.Data
Imports System.Text
Imports System.Windows.Forms

Namespace DevExpress.XtraRichEdit.Demos
	Partial Public Class BulletsAndNumberingModule
		Inherits DevExpress.XtraRichEdit.Demos.TutorialControl
		Public Sub New()
			InitializeComponent()
            RtfLoadHelper.Load("BulletsAndNumbering.rtf", richEditControl)
            CType(New RichEditDemoExceptionsHandler(richEditControl), RichEditDemoExceptionsHandler).Install()
			richEditControl.Focus()
		End Sub

		Public Overrides ReadOnly Property PrintingRichEditControl() As RichEditControl
			Get
				Return richEditControl
			End Get
		End Property

		Protected Overrides Sub DoShow()
			MyBase.DoShow()
			richEditControl.Focus()
		End Sub

	End Class
End Namespace

File: Components/WinForms/CS/RichEditMainDemo/Modules/BulletsAndNumbering.cs
using System;

namespace DevExpress.XtraRichEdit.Demos {
    public partial class BulletsAndNumberingModule : DevExpress.XtraRichEdit.Demos.TutorialControl {
        public BulletsAndNumberingModule() {
            InitializeComponent();
            RtfLoadHelper.Load("BulletsAndNumbering.rtf", richEditControl);
            richEditControl.Focus();
            new RichEditDemoExceptionsHandler(richEditControl).Install();
        }

        public override RichEditControl PrintingRichEditControl { get { return richEditControl; } }

        protected override void DoShow() {
            base.DoShow();
            richEditControl.Focus();
        }

    }
}
